python/lsst/ctrl/mpexec/separablePipelineExecutor.py

```diff
@@ -257,6 +256,10 @@ def run_pipeline(
         graph_executor : `lsst.ctrl.mpexec.QuantumGraphExecutor`, optional
             A custom graph executor. By default, a new instance of
             `lsst.ctrl.mpexec.MPGraphExecutor` is used.
+        num_proc : `int`, optional
+            The number of processes that can be used to run the pipeline. The
+            default value ensures that no subprocess is created. To determine
+            the number of available cores use `multiprocessing.cpu_count`.
```

Comment:
Please include the same "only with default executor" disclaimer as for `fail_fast`. The documentation for `fail_fast` itself should also be reworded to account for the fact that execution is no longer parallel by default.

Reply:
I'd recommend dropping the comment about `multiprocessing.cpu_count`; it's not relevant to this method. (Put another way, why bring it up it here, and not in all the other places where users specify number of processes?)